db/migrate/20230117042659_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).  In order to show that, the  `'author_id'` reference the User class. Kindly write the query like this so the foreign key name will be `author_id`:

```ruby
add_reference :posts, :author, null: false, foreign_key: { to_table: :users }
```

- Note: As you can see on the ERD diagram, they are using the `author_id` column as a foreign key referring to the User class and not the `user_id` column like you did, so that's why we need to build our migration to refer to the author_id column.